### ISSUE TITLE
Identify dropdown is not working for GeoStory map configuration

### ISSUE THREAD

**Issue by allyoucanmap** [GeoNode/geonode-mapstore-client#1275]
Steps to reproduce:

- create a new geostory
- change media source of the cover
- select a map from the resource list and apply it
- edit map configuration
- select settings tab

Expected result:

We can probably hide the dropdown and keep only enable/disable identify. This need to be discussed with @giohappy 

Current result:

The dropdown selection has not effect on the GFI

![image](https://user-images.githubusercontent.com/19175505/199044714-ff5d75a5-3e8c-41fc-8ec1-3ec01a2b759e.png)

**Comment by giohappy** [comment]
@allyoucanmap I agree, with should remove the dropdown. GFI should use the template from GeoNode, as we do for map resources, but for the moment we can just hide the selection.
